lago/cmd.py

```diff
@@ -271,6 +271,39 @@ def do_stop(prefix, vm_names, **kwargs):
     prefix.stop(vm_names=vm_names)
 
 
+@lago.plugins.cli.cli_plugin(help='Export virtual entities')
+@lago.plugins.cli.cli_plugin_add_argument(
+    '--entities-names',
+    '-e',
+    help='Name of the entities to export. If no name is specified, export all'
+    'the entities in this prefix',
+    metavar='ENTITY_NAME',
```

Comment:
1. about the meaning of 'entity' - from reading this I'm not sure what it does. 
2. I think here is the place to put more docs about what this command does:
``it exports all VMs into a ..., if standalone is used ....`` etc